lms/djangoapps/django_comment_client/management/management_utils.py

```diff
@@ -0,0 +1,49 @@
+'''
+This module provides utility function for the utility functions for django_comment_client
+'''
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+import logging
+from django.db import IntegrityError
+from pymongo.errors import PyMongoError
+
+log = logging.getLogger(__name__)
+
+
+def rename_user(old_username, new_username):
+    '''
+    This function changes the username of a user
+    :param old_username: old username of the user
+    :param new_username: new username desired for this user
+    :return: result {'status': 'success'/'failure', 'message': 'relevant message'}
+    '''
+    result = {'status': 'failure', 'message': ''}
+    try:
+        user = User.objects.get(username=old_username)
+    except User.DoesNotExist:
+        result['message'] = "User %s does not exist." % old_username
+        log.exception(result['message'])
+        return result
+    try:
+        user.username = new_username
+        user.save()
+    except IntegrityError:
+        result['message'] = 'Error modifying the username for %s. Rename not performed.' % old_username
+        log.exception(result['message'])
+        return result
+    else:
+        try:
+            client = MongoClient()
+            db = client.forum
+            db.users.update({'username': old_username}, {'$set': {'username': new_username}}, multi=True)
+            db.contents.update({'author_username': old_username}, {'$set': {'author_username': new_username}}, multi=True)
+        except PyMongoError:
+            result['message'] = 'Error modifying the username for %s in mongodb. Rolling back change.' % old_username
+            log.exception(result['message'])
+            user.username = old_username
+            user.save()
+            return result
+
+        result['status'] = 'success'
+        result['message'] = "changed username of user: %s to %s" % (old_username, new_username)
```

Comment:
Throughout our codebase, we use `str.format` instead of `%`-based string
interpolation.

```python
result['message'] = "changed username from '{username_old}' to '{username_new}'".format(
    username_old=old_username,
    username_new=new_username,
)
```